thunder/tests/test_jit_general.py

```diff
@@ -1205,6 +1206,13 @@ def forward(self, x):
     assert jitted.l1.weight.grad is not None
 
 
+@contextmanager
+def suppress_warnings():
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", message="Please use `torch.vmap`")
+        yield
+
+
```

Comment:
I think the same thing could be accomplished more succinctly with the annotation `@pytest.mark.filterwarnings("ignore:Please use `torch.vmap`")`.

Reply:
Yeah, makes sense. I just added these because Mike said:

> Maybe we should just wrap the gradcheck function in our own helper that suppresses this warning?

But I'll fix this with the decorator. Thank you!